src/XrdXrootd/XrdXrootdConfig.cc

```diff
@@ -1441,6 +1443,15 @@ int XrdXrootdProtocol::xmon(XrdOucStream &Config)
                    if (mrType) monRBval = static_cast<int>(tempval);
                       else     monMBval = static_cast<int>(tempval);
                   }
+         else if (!strcmp("fbuff", val))
+                {if (!(val = Config.GetWord()))
+                    {eDest.Emsg("Config", "monitor fbuff value not specified");
+                     return 1;
+                    }
+                 if (XrdOuca2x::a2sz(eDest,"monitor fbuff", val,
+                                             &tempval, 1024, 65536)) return 1;
```

Comment:
The upper limit should be 65472 not 65536 because it anyone specifies that the packet length in the header will become zero as it's a short. The lower limit should ideally be no less than 4096. If you network drops 4K datagram packets something is really wrong with your network.

Reply:
I agree that networks suck.  There are many articles on why fragmented UDP packets are difficult to use.  But, standard MTU is normally 1500, so just under 1500 usable.  We want to give the option to have smaller than MTU packets, to stop most fragmentation.